doc/fwutil/fwutil.md

```diff
@@ -340,7 +340,7 @@ The purpose of the update commands group is to provide an interface
 for automatic FW installation of various platform components.
 
 Automatic FW installation requires platform_components.json to be created and placed at:  
-_sonic-buildimage/device/<platform_name>/<onie_platform>/platform_components.json_
+_sonic-buildimage/device/<platform_name>/<onie_platform>/platform_fw_updatecomponents.json_
```

Comment:
@sujinmkang Typo? Maybe it's better to have something like `platform_fw_updates`?